pkg/pay/interface.go

```diff
@@ -0,0 +1,111 @@
+package pay
+
+import (
+	"fmt"
+
+	"github.com/stripe/stripe-go/v74"
+)
+
+type PaymentStatus string
+
+const (
+	PaymentSuccess    PaymentStatus = "success"
+	PaymentNotPaid    PaymentStatus = "not_paid"
+	PaymentProcessing PaymentStatus = "processing"
+	PaymentExpired    PaymentStatus = "expired"
+	PaymentFailed     PaymentStatus = "failed"
+	PaymentUnknown    PaymentStatus = "unknown"
+)
+
+const (
+	SuccessURL string = "https://cloud.sealos.io"
+	CancelURL  string = "https://cloud.sealos.io"
```

Comment:
Do not define consts in interface.go, add another file such as `consts.go` and name these consts with `default` prefix to distinguish.